package.json

```diff
@@ -21,5 +21,8 @@
     "lerna": "^8.1.2",
     "lint-staged": "^15.2.2",
     "prettier": "^3.2.5"
+  },
+  "dependencies": {
+    "uuid": "^11.0.2"
```

Comment:
If this is just needed for tests it should be a devDependency, something like this:

```
npm uninstall uuid
```

and
```
npm install -D uuid
```

Reply:
Changed